egs/aspire/s5/local/nnet3/run_blstm.sh

```diff
@@ -89,9 +89,9 @@ if [ $stage -le 7 ]; then
   fast-lstmp-layer name=blstm2-backward input=Append(blstm1-forward, blstm1-backward) delay=2 $lstm_opts
 
   fast-lstmp-layer name=blstm3-forward input=Append(blstm2-forward, blstm2-backward) delay=-3 $lstm_opts
-  fast-lstmp-layer name=blstm3-backward input=Append(blstm2-forward, blstm2-backward delay=3 $lstm_opts
+  fast-lstmp-layer name=blstm3-backward input=Append(blstm2-forward, blstm2-backward) delay=3 $lstm_opts
 
-  output-layer name=output output-delay=$label_delay dim=$num_targets max-change=1.5
+  output-layer name=output input=Append(blstm3-forward, blstm3-backward) output-delay=$label_delay dim=$num_targets max-change=1.5
```

Comment:
I'm wondering what does the label delay mean in a bidi network. It's likely intended to be 0 (and, generally, throughout the whole script). What sense does it make?

@jtrmal, do you have experience with the BLSTM training? Also, `include-log-softmax` is `true` by default. PTAL.

```suggestion
  output-layer name=output input=Append(blstm3-forward, blstm3-backward) include-log-softmax=false output-delay=0 dim=$num_targets max-change=1.5
```

Reply:
I'm actually not sure. @danpovey  could you please help?

Reply:
0 is the default so it's not really important here.

Reply:
@danpovey I'd rip it out. It's only confusing. But since the recipe predates CTC... Maybe whatever.

Reply:
@danpovey And `include-log-softmax=false` is needed here, right?

Reply:
we normally use include-log-softmax=false for the output layer for MPE, it saves a little time although it doesn't affect the result.

Reply:
.. also: output-delay should be 0 for blstm, yes.